source/jormungandr/jormungandr/interfaces/v1/Schedules.py

```diff
@@ -162,6 +168,31 @@ def __init__(self, endpoint, *args, **kwargs):
     def options(self, **kwargs):
         return self.api_description(**kwargs)
 
+    def _add_direction_type_filter(self, direction_type, filter):
+
+        # (forward, clokwise, inbound) are equivalent and
+        # (backward, anticlokwise, outbound) are equivalent too.
+        def create_direction_type_filter(direction_type):
+            if direction_type == 'forward':
+                values = 'forward,clockwise,inbound'
+            elif direction_type == 'backward':
+                values = 'backward,anticlockwise,outbound'
+            elif direction_type == 'all':
+                values = 'all'
```

Comment:
In that case, you will return only the routes whose direction_type is `all` no?

Reply:
For the `all` case, we don't need to create a filter. 
This is the case with https://github.com/CanalTP/navitia/pull/2967/files#diff-71deae639b1c21f792ab41c775252465R238
This case is just for being logical into the **create_direction_type_filter** function, in case of we'll use it elsewhere.

Reply:
Okay, I understood your catch. If we create, a filter like `route.has_direction_type(all)`, it is not good, you have right. But it is not possible into this code because I have protected with `all` before.

I just have to remove the `elif direction_type == "all"` for fix that. It will be more clear. 
Thanks for the catch.